.gitlab/pipeline/short-benchmarks.yml

```diff
@@ -24,3 +27,58 @@ short-benchmark-westend:
   <<: *short-bench
   variables:
     RUNTIME: westend
+
+# run short-benchmarks for system parachain runtimes from cumulus
+
+.short-benchmark-cumulus: &short-bench-cumulus
```

Comment:
`.short-benchmark-cumulus` and `.short-benchmark-polkadot` could be possibly merged to `.short-benchmark-template` with some variables parametrization, do we want to do it?